src/main/java/org/nantipov/utils/wordhugopress/config/job/convertposts/ConvertPostsReaderConfig.java

```diff
@@ -25,25 +25,55 @@
 @Configuration
 public class ConvertPostsReaderConfig {
 
-    private static final String SQL_QUERY_POSTS =
-            "SELECT\n" +
+    private static final Set<String> DRAFT_STATUSES = ImmutableSet.of("draft", "auto-draft");
+
+    static final String BEAN_NAME = "postsReader";
+
+    private final SourcesSettings sourcesSettings;
+    
+    public ConvertPostsReaderConfig(SourcesSettings sourcesSettings) {
+        this.sourcesSettings = sourcesSettings;
+    }
+    
+    private static String TABLE_PREFIX;
+
+    private static String SQL_QUERY_POSTS; 
```

Comment:
1. Java naming convention. Snake uppercased identifiers are supposed to be used with `static final` only.
2. Variables `TABLE_PREFIX` and `SQL_QUERY_POSTS` could be local by their involvement in the code (used in a single method).